### PR TITLE
Add parm over ssh config and enable LDAP authentication

### PR DESCRIPTION
**Pull request by mboussaa**

### CODE REVIEW

File: proactive-installer/installer-and-healthcheck/inputConfig.json [hunk 1]
```diff
@@ -1,64 +1,82 @@
...
     "passphrase": "changeme",
     "libraries": "rsync,wget,curl,zip,unzip,sshpass,openssl,gpg,passwd",
     "installationDirectory": "/opt/proactive",
     "systemUser": "proactive",
     "systemUserGroup": "proactive",
     "systemUserPassword": "jA0ECQMCMH3ZzJzCY2Pu0j4BYAnaCM68MbjvX5ltAbdOIol3pJI6lHLM1pfCW2w0dYFyOdFRF4KDhUy3KxdQ6dOHLKsWWHWccG4tHV627g==",
     "networkProtocol": "pamr",
-    "webProtocol": "https",
-    "webPort": "8443",
+    "webProtocol": "http",
```
Comment (nebilbenmabrouk) [inline proactive-installer/installer-and-healthcheck/inputConfig.json:11]: why http ?

File: proactive-installer/installer-and-healthcheck/inputConfig.json [hunk 2]
```diff
@@ -1,64 +1,82 @@
...
+    "userSubtree" : "ou=users,dc=activeeon-microsoft,dc=org",
+    "groupSubtree" : "ou=groups,dc=activeeon-microsoft,dc=org",
+    "bindLogin" : "cn=admin,dc=activeeon-microsoft,dc=org",
+    "bindPassword" : "jA0ECQMCMlIuxtErGLL30kMB2IdRpj+e299SW6h79aqmiPtAD9CsgEf9TwsH6LBoRbLEcZXwFcMn8TCxn0gZ0eWhkCQPAZ1DNnvJcSBC1yn0at9I",
+    "userFilter" : "(&(objectclass=inetOrgPerson)(uid=%s))",
+    "groupFilter" : "(&(objectclass=groupOfUniqueNames)(uniqueMember=%s))",
+    "testLogin" : "viale",
+    "testPassword" : "jA0ECQMC64bQy3YO0UX30kEB8VKsVq35OgP/fgZqyaRbuDjU9KPC2vZ+wNDf9t6aC2PiJGZlgfqcWhCoH91yp9ufkPS7DHZ7zTPvoT1AOdYQbQ==",
+    "adminRoles" : "activeeon-admins",
+    "userRoles" : "testGroup"
```
Comment (nebilbenmabrouk) [inline proactive-installer/installer-and-healthcheck/inputConfig.json:34]: You should replace the values with fake simple values

**Review by mbenguig — APPROVED** [review]
Seems "globally" ok.

File: proactive-installer/installer-and-healthcheck/inputConfig.json (hunk 1 above)
Comment (mboussaa) [reply proactive-installer/installer-and-healthcheck/inputConfig.json:11]: because there is no certficate

**Pull request merged**